ai/cryptocom-ai-agent-chatbot/src/integration/chain-ai.api.ts

```diff
@@ -32,13 +32,6 @@ export const chainAiInstance = {
           openAI: {
             apiKey: import.meta.env.VITE_OPEN_AI_KEY,
           },
-          chain: 282,
```

Comment:
why do we not need the chain id now?

Reply:
Or is this just an example because the [backend already has the chainid](https://github.com/crypto-com/developer-platform-sdk-examples/blob/chatbot-tutorial/ai/cryptocom-ai-agent-service/src/services/agent/agent.service.ts#L32)?